packages/contracts-bedrock/src/libraries/Pausable.sol

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Why not just use the Pausable from the openzeppelin package? How is this different from the original Pausable contract?